test/python/circuit/test_unitary.py

```diff
@@ -296,3 +296,10 @@ def test_unitary_decomposition_via_definition_2q(self):
         """Test decomposition for 2Q unitary via definition."""
         mat = numpy.array([[0, 0, 1, 0], [0, 0, 0, -1], [1, 0, 0, 0], [0, -1, 0, 0]])
         numpy.allclose(Operator(UnitaryGate(mat).definition).data, mat)
+
+    def test_unitary_control(self):
+        """Test parameters of controlled - unitary."""
+        mat = numpy.array([[0, 1], [1, 0]])
+        gate = UnitaryGate(mat).control()
+        numpy.allclose(gate.params, mat)
```

Comment:
Does this need to be in a `self.assertTrue` call to raise an error if it's not all close?

Reply:
@georgios-ts I think this is the only comment left then we can merge 🙂